ui/keyboardMove/src/plugins/keyboardMove.ts

```diff
@@ -90,6 +90,11 @@ export default (opts: Opts) => {
         opts.ctrl.draw();
         clear();
       }
+    } else if (v.length > 0 && 'next'.startsWith(v.toLowerCase())) {
+      if ('next' === v.toLowerCase()) {
+        opts.ctrl.next && opts.ctrl.next();
```

Comment:
`opts.ctrl.next?.()` is a simpler way to write this

Reply:
This is so nice, and I always forget to use it for some reason. Maybe I will finally remember going forward 🤞